src/parseTools.js

```diff
@@ -838,7 +838,7 @@ function hasExportedSymbol(sym) {
 // it is a BigInt. Otherwise, we legalize into pairs of i32s.
 function defineI64Param(name) {
   if (WASM_BIGINT) {
-    return `/** @type {!BigInt} */ ${name}`;
```

Comment:
This was removed because otherwise closure generates errors like this one:

```
/tmp/emscripten_temp_16el3cc_/hello_world.js:20729:6: WARNING - [JSC_TYPE_MISMATCH] assignment
found   : number
required: BigInt
  20729|       timeout = Number(timeout);
```

This was happening for any function that used `receiveI64ParamAsI53Unchecked` which is currently just `glClientWaitSync` and `glWaitSync`.